tools/vsimporter/src/utils/xc2vs.cpp

```diff
@@ -95,7 +95,7 @@ static VCProjectItem* addFileToVSInternal(const PBXFile* file, VCProject& proj,
   // Take into account path override
   bool useRelativePath = true;
   if (itemHint && !itemHint->pathOverride.empty()) {
-    sbAssert(!isVariant, "Unexpected path override for variant file: " + realPath);
+    sbAssertWithTelemetry(!isVariant, "Unexpected path override for variant file: " + realPath);
```

Comment:
The path does contain PI, but it will be stripped off since it comes after the colon.

---
In reply to: [82424365](https://github.com/Microsoft/WinObjC/pull/1104#discussion_r82424365) [](ancestors = 82424365)